src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+@import "../utils/mixins";
+@import "../utils/veriables";
+@import "../utils/extands";
+
+.calendar {
+  display: flex;
+  justify-content: center;
+
+  &__list {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: ($square-size * 7) + 10px;
+    margin-top: 21px;
+    gap: $gap-size;
+  }
+
+  &__day {
+    display: flex;
+    @include square($square-size);
+    justify-content: center;
+    align-items: center;
+    border: 1px solid black;
+
+    background-color: #eee;
+
+    @extend %date-text;
+    list-style: none;
+
+    cursor: pointer;
+
+    &:hover {
+      background-color: pink;
+
+      transform: translateY(-20px);
+      transition: 0.5s;
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__day--#{$i}::before {
+      content: "#{$i}";
+      position: absolute;
+    }
+  }
+
+  @each $value, $week-day in $week-days {
+    &__day--start-day-#{$week-day} {
+      margin-left: ($square-size * (-1 + $value)) + ($gap-size * (-1 + $value));
+    }
+  }
```

Comment:
Implement **`start-day`** modifier for the **`calendar`**, this should be a modifier for calendar, and you should add margin for the first day, use corresponding pseudo-class for it